packages/lib/storybook/stories/helpers/RedirectResultContainer.tsx

```diff
@@ -1,21 +1,33 @@
 import { useEffect, useState } from 'preact/hooks';
 import { AdyenCheckout } from '../../../src';
 import { handleError, handleFinalState } from '../../helpers/checkout-handlers';
+import { AdditionalDetailsStateData } from '../../../src/types/global-types';
+import { makeDetailsCall } from '../../helpers/checkout-api-calls';
 
-export const RedirectResultContainer = ({ redirectResult, sessionId }) => {
+export const RedirectResultContainer = ({ redirectResult, sessionId, countryCode }) => {
     const [isRedirecting, setIsRedirecting] = useState<boolean>(true);
     let message = isRedirecting ? 'Submitting details...' : '';
 
     useEffect(() => {
-        if (!redirectResult || !sessionId) {
+        if (!redirectResult && !sessionId) {
             message = 'There is no redirectResult / sessionId provided';
             return;
         }
 
         AdyenCheckout({
             clientKey: process.env.CLIENT_KEY,
             environment: process.env.CLIENT_ENV,
-            session: { id: sessionId },
+            ...(sessionId && { session: { id: sessionId, countryCode } }),
```

Comment:
Here we already have the session id but we have to pass the country code otherwise it won't work. Failed at `initialize` function.